cmd/ci-secret-generator/main.go

```diff
@@ -57,6 +58,7 @@ func parseOptions() options {
 	flag.CommandLine.StringVar(&o.configPath, "config", "", "Path to the config file to use for this tool.")
 	flag.CommandLine.StringVar(&o.bootstrapConfigPath, "bootstrap-config", "", "Path to the config file used for bootstrapping cluster secrets after using this tool.")
 	flag.CommandLine.BoolVar(&o.validate, "validate", true, "Validate that the items created from this tool are used in bootstrapping")
+	flag.CommandLine.BoolVar(&o.validateOnly, "validate-only", false, "If the tool should exist after the validation")
```

Comment:
`s/exist/exit/`

Reply:
But it will stop existing after the validation!

Reply:
updated